src/components/add-cash/AddCashFooter.js

```diff
@@ -1,14 +1,15 @@
 import PropTypes from 'prop-types';
 import React, { useCallback } from 'react';
+import { useNavigation } from 'react-navigation-hooks';
 import { useDimensions } from '../../hooks';
 import { colors } from '../../styles';
+import { ButtonPressAnimation } from '../animations';
 import { Centered, ColumnWithMargins, Row, RowWithMargins } from '../layout';
 import { Emoji, Text } from '../text';
 import ApplePayButton from './ApplePayButton';
-import { ButtonPressAnimation } from '../animations';
-import { useNavigation } from 'react-navigation-hooks';
 
 const AddCashFooter = ({ disabled, onDisabledPress, onSubmit, ...props }) => {
+  const { isTallPhone } = useDimensions();
   const { isTinyPhone } = useDimensions();
```

Comment:
Can combine these two lines to: const { isTallPhone, isTinyPhone } = useDimensions();